.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: python
+
+python:
+  - "3.5"
```

Comment:
Is this actually testing each version of python? It looks like the different django and drf versions are being installed properly but it looks like everything is against 3.5 in the travis logs? https://travis-ci.org/yeti/YAK-server/jobs/118313043